Tests/headers/devicesettings/securityagent/SecurityTokenUtil.h

```diff
@@ -0,0 +1,2 @@
+#pragma once
+int GetSecurityToken(unsigned short maxLength, unsigned char buffer[]);
```

Comment:
please remove

Reply:
this one is left with default implementation due to reference in utils.cpp, we will remove it later